LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2014, Google Inc. All rights reserved.
+Copyright 2022 Google LLC. All rights reserved.
```

Comment:
Per Google's internal recommendation, we should retain the earliest year:

```
When making changes to code with an existing copyright notice:

- Retain earliest copyright year.

- Update Google Inc. to Google LLC

- Optionally, add current copyright year.
```

See https://g3doc.corp.google.com/company/teams/opensource/copyright.md?cl=head for details.

Reply:
Ah, I see.  I misunderstood before.  I'll close this and replace it with a PR that only updates the company name.